packages/components/input-number/src/input-number.vue

```diff
@@ -161,54 +165,50 @@ export default defineComponent({
       }
       return precision
     }
-    const _increase = (val: number) => {
-      if (!isNumber(val)) return data.currentValue
-      const precisionFactor = 10 ** numPrecision.value
-      // Solve the accuracy problem of JS decimal calculation by converting the value to integer.
-      val = isNumber(val) ? val : Number.NaN
-      return toPrecision(
-        (precisionFactor * val + precisionFactor * props.step) / precisionFactor
-      )
-    }
-    const _decrease = (val: number) => {
+    const ensurePrecision = (val: number, coefficient: 0 | 1 | -1 = 1) => {
```

Comment:
**Question:**  `0` was never used, then why are we adding it in to the type of `coefficient`